Makefile

```diff
@@ -17,6 +17,10 @@ PROJECT := kilo
 PKG := github.com/squat/$(PROJECT)
 REGISTRY ?= index.docker.io
 IMAGE ?= squat/$(PROJECT)
+ifneq ($(REGISTRY),index.docker.io)
```

Comment:
Does pushing to docker hub break if you prefix the manifest names with the registry? If it doesn't break, then let's avoid this conditional. I see that we can push images that are prefixed just fine.

I'm wondering if we really need the this new variable, which is conditionally prefixed or not and is complex in my mind, or we can get away with directly prefixing all occurrences of `$(IMAGE)` with `$(REGISTRY)/`

Reply:
Done, by the way, I'm also sorry it too me so long to get back to you.